GOVERNANCE.md

```diff
@@ -165,6 +174,84 @@ We have a reasonably liberal approach to code review and merging. We value quick
 - All code must go through Pull Requests (PR) and pass status checks before being merged. If a PR is merged that breaks `main` due to the branch not being up-to-date, then it should either be reverted or a quick fix merged as a separate PR.
 - If a PR is against code you have previously committed and is either small changes, bug fixes, or refactors, then you're free to merge it without any review. However, if you need more confidence in your changes, you can wait for approval from another maintainer or core contributor.
 
+## Financial Contributions
+
+Biome is in the fortunate position that people care enough about it that they want to invest financially in its future. However, with money comes responsibility, so we are committed to the following.
+
+There are two ways to fund Biome and its development: sponsorship and funded bounties, although we strongly prefer sponsorship.
+
+### Sponsorship
+
+Sponsorship is the first and foremost way of making financial contributions to Biome. Sponsorship can be
+done through a one-time donation, or through recurring donations. We offer incentives for certain levels of recurring donations.
+
+To learn more about sponsorship incentives, or to make a donation, please visit our [Open Collective page](https://opencollective.com/biome).
+You can also sponsor the project through [GitHub](https://github.com/sponsors/biomejs).
+
+### Community-Funded Bounties
```

Comment:
I'm not sure if there's a better alternative, but the words "issue" and "project" are already used for units of work, which may or may not be funded. The word "bounty" unambiguously clarifies there's a financial reward for a unit of work. The Polar terminology makes sense within their realm, since the platform is all about funding, so they differentiate "issues" from other financial transactions, but applying that terminology to Biome would be more confusing than helpful, I think.

I don't think there's an implication that bounties should only be for small things. That should be reflected in the pledged amount instead. But I do think issues with a bounty attached shouldn't become *too* big, hence the $1000 maximum. If a task is so large we feel $1000 doesn't cover the effort, it would be better to split it into multiple tasks with separate bounties.

Reply:
Community-funded issue?

Reply:
I can change the header, yes. But the word bounty is used all-over the place in this doc, and using the full phrase "Community-funded issue" everywhere seems quite burdensome. Assuming we don't want to ban the word bounty altogether,  I'm not a fan of changing the header either, since it would only create a disconnect between the terminology used.